packages/terra-toolbar/src/Toolbar.jsx

```diff
@@ -0,0 +1,40 @@
+import React from 'react';
+import PropTypes from 'prop-types';
+import classNames from 'classnames/bind';
+import styles from './Toolbar.module.scss';
+
+const cx = classNames.bind(styles);
+
+const propTypes = {
+  /**
+   * Content alignment. Align to start, end, or center.
+   */
+  align: PropTypes.oneOf(['start', 'end', 'center']),
+  /**
+   * Items to be displayed in toolbar.
+   */
+  children: PropTypes.node,
```

Comment:
How are we approaching responsiveness for the toolbar? Are we expecting CollapsibleMenuViews as children here? Should be be wrapping the flex items in the toolbar vs. shrinking them on a single line?

Reply:
For this version of the toolbar we don't have responsive behavior. Collapsible menu view is not supported and it's consumer's responsibility not to put too much into it that it can't all fit on one line. There will be other variants in the future that will handle collapsible menu view.

Reply:
Could we wrap?

Reply:
If we allow the items to wrap, then items that have long content will push the items to wrap before it will wrap its' own content. 
Wrap:
![Long_Content default](https://user-images.githubusercontent.com/4084424/76120316-dde9e000-5fb6-11ea-9813-bc947fb5e030.png)
No wrap:
![Long_Content default](https://user-images.githubusercontent.com/4084424/76120387-007bf900-5fb7-11ea-962b-3c803e00d82b.png)

Reply:
Eventually we're going to have to support font scaling without breaking the page. To do that we're going to have to wrap this content. There also the concern of translations pushing content outside the bounds of the toolbar.

I think we have to wrap.

Reply:
+1 to wrapping.  Let's make sure we discuss it with Seth as well though